File: IronVault/UI/ironvault_mobile/lib/models/korisnik_nutricionist.dart
import 'package:ironvault_mobile/models/korisnik.dart';
import 'package:ironvault_mobile/models/nutricionist.dart';
import 'package:json_annotation/json_annotation.dart';

part 'korisnik_nutricionist.g.dart';

@JsonSerializable()
class KorisnikNutricionist {
  int? korisnikNutricionistId;
  int? korisnikId;
  Korisnik? korisnik;
  int? nutricionistId;
  Nutricionist? nutricionist;
  DateTime? datumTermina;
  int? zakazanoSati;

  KorisnikNutricionist({this.korisnikNutricionistId});

  factory KorisnikNutricionist.fromJson(Map<String, dynamic> json) =>
      _$KorisnikNutricionistFromJson(json);

  /// Connect the generated [_$PersonToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$KorisnikNutricionistToJson(this);
}

File: IronVault/IronVault.Model/Models/KorisnikNutricionist.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace IronVault.Model.Models
{
    public partial class KorisnikNutricionist
    {
        public int KorisnikNutricionistId { get; set; }

        public int KorisnikId { get; set; }

        public int NutricionistId { get; set; }

        public DateTime DatumTermina { get; set; }

        public int ZakazanoSati { get; set; }

        public virtual Korisnik Korisnik { get; set; } = null!;

        public virtual Nutricionist Nutricionist { get; set; } = null!;
    }
}
